doc/releases/changelog-dev.md

```diff
@@ -21,6 +21,8 @@
 * `qml.draw` and `qml.draw_mpl` now render operator ids.
   [(#4749)](https://github.com/PennyLaneAI/pennylane/pull/4749)
 
+* `default.qutrit` is now differentiable via backprop using TRX, TRY, and TRZ
```

Comment:
```suggestion
* `default.qutrit` is now differentiable via backprop using TRX, TRY, and TRZ
  [(#4790)](https://github.com/PennyLaneAI/pennylane/pull/4790)
```
If you would like, don't forget to add yourself as a contributor at the bottom of this doc!